### PR TITLE
Log in & remember in session

### PR DESCRIPTION
**Pull request by martijnjanssen**

### CODE REVIEW

File: src/lancie-admin-app.html [hunk 1]
```diff
@@ -18,17 +20,64 @@
...
+        user: Object,
+        token: String,
+        loggedIn: {
+          type: Boolean,
+          value: false,
+        },
+      },
+
+      attached: function() {
+        var token = window.sessionStorage.getItem('auth-token');
```
Comment (TimvdLippe) [inline src/lancie-admin-app.html]: Just wondering: why sessionStorage here and not local storage? Can use the same storage also such that admins do not have to login separately here.

File: src/lancie-admin-app.html [hunk 2]
```diff
@@ -18,17 +20,64 @@
...
+        if (!!token && token !== '') {
```
Comment (TimvdLippe) [inline src/lancie-admin-app.html]: I think the `token !== ''` is unnecessary. Since `!!'' === false`

File: src/lancie-admin-app.html [hunk 3]
```diff
@@ -18,17 +20,64 @@
...
+      storeLogin: function(e, detail) {
+        if (detail.user.authorities.indexOf('ROLE_ADMIN') > -1) {
+          this.user = detail;
+          this.$.ajax.injectToken(detail.token);
+          window.sessionStorage.setItem('auth-token', detail.token);
+
+          this.loggedIn = true;
+        } else {
+          // TODO: Raplace with toast
+          alert('You are not an admin!');
```
Comment (TimvdLippe) [inline src/lancie-admin-app.html]: For consistency use `window.alert`

**Review by TimvdLippe — DISMISSED** [review]
Statically reviewed and seems fine. Have not actually tested it as I am not an admin myself.

File: src/lancie-admin-app.html (hunk 1 above)
Comment (martijnjanssen) [reply src/lancie-admin-app.html]: Good point, I will change the areafiftylan frontend over to the localstorage soon. This implementation (no app-localstorage-document) is easier in my opinion.

File: bower.json [hunk 4]
```diff
@@ -11,7 +11,9 @@
     "wisvch-footer": "wisvch/wisvch-footer#^1.0.1",
     "app-layout": "PolymerElements/app-layout#^0.10.4",
     "app-route": "polymerelements/app-route#^0.9.2",
-    "iron-lazy-pages": "TimvdLippe/iron-lazy-pages#2.0-preview"
+    "iron-lazy-pages": "TimvdLippe/iron-lazy-pages#2.0-preview",
+    "lancie-login-card": "AreaFiftyLAN/lancie-login-card#1.0.0",
+    "lancie-ajax": "AreaFiftyLAN/lancie-ajax#1.0.0"
```
Comment (MatthijsKok) [inline bower.json:16]: Please use 1.0.1

Comment (MatthijsKok) [reply bower.json:16]: doesn't matter for this release but fixes future bugs

**Review by MatthijsKok — APPROVED** [review]
looking good

File: src/lancie-admin-app.html [hunk 5]
```diff
@@ -17,18 +20,108 @@
       lancie-content {
         @apply(--layout-flex);
       }
+
+      .login-header {
+        height: 64px;
+        background-color: #1a2b43;
```
Comment (FrancisBehnen) [inline src/lancie-admin-app.html]: Any particular reason to not use `var(--primary-color)`?

**Review by FrancisBehnen — APPROVED** [review]
Looks good and works

File: bower.json (hunk 4 above)
Comment (martijnjanssen) [reply bower.json:16]: #14 already makes that change.

**Pull request merged**